core/constants.py

```diff
@@ -0,0 +1,22 @@
+from enum import auto, Enum
+
+
+class Position(Enum):
```

Comment:
Should we call this something more explicit? Like SpeciesPosition? Or CardPosition?

Reply:
Yes.